src/crdt.rs

```diff
@@ -954,6 +955,7 @@ impl Crdt {
     }
     fn run_window_request(
         window: &SharedWindow,
+        ledger_window: &mut Option<&mut LedgerWindow>,
```

Comment:
Why is the `Option` `&mut`?

Reply:
it was the only way I could figure out how to get the ledgerwindow through a move and a loop

Reply:
if you're on the Rust server, you can look up my question in #beginners